internal/controller/ocivalidator_controller.go

```diff
@@ -53,18 +55,16 @@ func (r *OciValidatorReconciler) Reconcile(ctx context.Context, req ctrl.Request
 
 	validator := &v1alpha1.OciValidator{}
 	if err := r.Get(ctx, req.NamespacedName, validator); err != nil {
-		// Ignore not-found errors, since they can't be fixed by an immediate requeue
-		if apierrs.IsNotFound(err) {
-			return ctrl.Result{}, nil
+		if !apierrs.IsNotFound(err) {
+			r.Log.Error(err, "failed to fetch OciValidator")
```

Comment:
i slightly modified this from what was in the shared reference PR. IMO we shouldnt log errors everytime we delete a CR since thats a pretty standard action. let me know what you think @TylerGillson